### ISSUE TITLE
It compiles however does not open the window in win10

### ISSUE THREAD

**Issue by jeffotoni** [lxn/walk#405]
What do I need to do to work on win10?

My example is test.go and I added

package main

// + build -ldflags = "- H windowsgui"

And in the manifest I put <assemblyIdentity type = "win64" and type = "win64"

How do I proceed so that my .exe opens the win10?

**Comment by lxn** [comment]
Please make sure you have a corresponding manifest in place, as explained in the README.

**Comment by jeffotoni** [comment]
This here ?

<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
    <assemblyIdentity version="1.0.0.0" processorArchitecture="*" name="SomeFunkyNameHere" type="win32"/>
    <dependency>
        <dependentAssembly>
            <assemblyIdentity type="win32" name="Microsoft.Windows.Common-Controls" version="6.0.0.0" processorArchitecture="*" publicKeyToken="6595b64144ccf1df" language="*"/>
        </dependentAssembly>
    </dependency>
    <asmv3:application>
        <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
            <dpiAware>true</dpiAware>
        </asmv3:windowsSettings>
    </asmv3:application>
</assembly>

**Comment by lxn** [comment]
Yes, that's it. The README explains what to do with that to make it work.

**Comment by jeffotoni** [comment]
There is something wrong in the example, it compiles correctly, it generates the .syso from rsrc.exe but when we run it it does not open ANYTHING ... it gets no GUI.

But the compilation does it right, understand?

**Comment by lxn** [comment]
If you have embedded the manifest with rsrc and it still doesn't work, then please try the other option with the appropriately named manifest file next to your executable. Please read the README section about this carefully. You may need to recompile your executable to make it work. Windows is weird.

**Comment by jeffotoni** [comment]
Very strange, I did differently and it worked, cool that worked.

Thank you
